### DISCUSSION TITLE
what is the code to check if the player had not yet had their first kiss monika?

### DISCUSSION THREAD

**Discussion by CaniBeTheTweek** [Monika-After-Story/MonikaModDev#9097] [QA - code and content creation (technical)] [answered]
I don't understand lol do I just add in `rules=rules `? and if not then just do `unlocked=False`?

**Comment by CaniBeTheTweek** [comment]
pretty sure I should add `persistent._mas_first_kiss,` but what comes after that?

**Comment by Booplicate** [comment, marked as answer]
```renpy
if mas_shouldKiss(3):
    call monika_kissing_motion_short
```

**Reply by CaniBeTheTweek** [reply]
wait wha- 
that's for the animation right? oh oh or does the `if mas_shouldKiss(3):` already do the check

**Reply by Booplicate** [reply]
`mas_shouldKiss` does the check. Where `3` is the chance of the kiss (1/3).

**Reply by CaniBeTheTweek** [reply]
ohh okay tysm!

**Reply by ThePotatoGuy** [reply]
This is correct, but `_mas_first_kiss` is the definitive flag for whether or not the user has had a first kiss with Monika.
